src/Tasks/Microsoft.CSharp.CurrentVersion.targets

```diff
@@ -234,92 +234,80 @@ Copyright (C) Microsoft Corporation. All rights reserved.
-         EmbedAllSources="$(EmbedAllSources)"
-         EmbeddedFiles="@(EmbeddedFiles)"
-         EmitDebugInformation="$(DebugSymbols)"
-         EnvironmentVariables="$(CscEnvironment)"
-         ErrorEndLocation="$(ErrorEndLocation)"
-         ErrorLog="$(ErrorLog)"
-         ErrorReport="$(ErrorReport)"
-         Features="$(Features)"
-         InterceptorsPreviewNamespaces="$(InterceptorsPreviewNamespaces)"
-         FileAlignment="$(FileAlignment)"
-         GeneratedFilesOutputPath="$(CompilerGeneratedFilesOutputPath)"
-         GenerateFullPaths="$(GenerateFullPaths)"
-         HighEntropyVA="$(HighEntropyVA)"
-         Instrument="$(Instrument)"
-         KeyContainer="$(KeyContainerName)"
-         KeyFile="$(KeyOriginatorFile)"
-         LangVersion="$(LangVersion)"
-         LinkResources="@(LinkResource)"
-         MainEntryPoint="$(StartupObject)"
-         ModuleAssemblyName="$(ModuleAssemblyName)"
-         NoConfig="true"
-         NoLogo="$(NoLogo)"
-         NoStandardLib="$(NoCompilerStandardLib)"
-         NoWin32Manifest="$(NoWin32Manifest)"
-         Nullable="$(Nullable)"
-         Optimize="$(Optimize)"
-         Deterministic="$(Deterministic)"
-         PublicSign="$(PublicSign)"
-         OutputAssembly="@(IntermediateAssembly)"
-         OutputRefAssembly="@(IntermediateRefAssembly)"
-         PdbFile="$(PdbFile)"
-         Platform="$(PlatformTarget)"
-         Prefer32Bit="$(Prefer32Bit)"
-         PreferredUILang="$(PreferredUILang)"
-         ProjectName="$(MSBuildProjectName)"
-         ProvideCommandLineArgs="$(ProvideCommandLineArgs)"
-         References="@(ReferencePathWithRefAssemblies)"
-         RefOnly="$(ProduceOnlyReferenceAssembly)"
-         ReportAnalyzer="$(ReportAnalyzer)"
-         ReportIVTs="$(ReportIVTs)"
-         Resources="@(_CoreCompileResourceInputs);@(CompiledLicenseFile)"
-         ResponseFiles="$(CompilerResponseFile)"
-         RuntimeMetadataVersion="$(RuntimeMetadataVersion)"
-         SharedCompilationId="$(SharedCompilationId)"
-         SkipAnalyzers="$(_SkipAnalyzers)"
-         SkipCompilerExecution="$(SkipCompilerExecution)"
-         Sources="@(Compile)"
-         SubsystemVersion="$(SubsystemVersion)"
-         TargetType="$(OutputType)"
-         TargetFramework="$(TargetFramework)"
-         ToolExe="$(CscToolExe)"
-         ToolPath="$(CscToolPath)"
-         TreatWarningsAsErrors="$(TreatWarningsAsErrors)"
-         UseHostCompilerIfAvailable="$(UseHostCompilerIfAvailable)"
-         UseSharedCompilation="$(UseSharedCompilation)"
-         Utf8Output="$(Utf8Output)"
-         VsSessionGuid="$(VsSessionGuid)"
-         WarningLevel="$(WarningLevel)"
-         WarningsAsErrors="$(WarningsAsErrors)"
-         WarningsNotAsErrors="$(WarningsNotAsErrors)"
-         Win32Icon="$(ApplicationIcon)"
-         Win32Manifest="$(Win32Manifest)"
-         Win32Resource="$(Win32Resource)"
-         PathMap="$(PathMap)"
-         SourceLink="$(SourceLink)">
```

Comment:
These feel like they belong only in final outputs so no need to pass them here, right?